x-pack/plugins/ml/server/models/data_recognizer/modules/security_auth/ml/datafeed_auth_high_count_logon_events_for_a_source_ip.json

```diff
@@ -5,16 +5,26 @@
   ],
   "max_empty_searches": 10,
   "query": {
-        "bool": {
-            "filter": [{"exists": {"field": "source.ip"}}],
-            "must": [
-                {"bool": {
-                    "should": [
-                    {"term": {"event.category": "authentication"}},
-                    {"term": {"event.outcome": "success"}}
-                    ]
-                }}
-            ]
+    "bool": {
+      "filter": [
+        {
+          "exists": {
+            "field": "source.ip"
+          }
         }
+      ],
```

Comment:
You could instead include everything in the `filter` clause itself. 

General rule:
Use `filter` when you don't care about document relevance, and use `must` when you do.

See this for more info: https://www.elastic.co/guide/en/elasticsearch/reference/current/query-filter-context.html

Reply:
Thanks for pointing, made the changes as suggested.